scripts/shortint-tests.sh

```diff
@@ -111,15 +111,16 @@ and not test(~smart_add_and_mul)"""
         --cargo-profile "${cargo_profile}" \
         --package "${tfhe_package}" \
         --profile ci \
-        --features="${ARCH_FEATURE}",shortint,internal-keycache,experimental \
+        --features="${ARCH_FEATURE}",shortint,internal-keycache,zk-pok,experimental \
```

Comment:
Why activate the zk-pok feature?

Reply:
because it's enabled above and without it we are recompiling for no good reason